packages/slice-machine/components/DeleteDocumentsDrawer/DeleteDocumentsDrawerOverLimit.tsx

```diff
@@ -38,6 +37,24 @@ const DeleteDocumentsDrawerOverLimit: React.FunctionComponent = () => {
     return null;
   }
 
+  const associatedDocumentsCards = modalData.details.customTypes.map(
+    (customTypeDetail) => {
+      const customType = availableCustomTypes.find(
+        (customType) => getModelId(customType) === customTypeDetail.id
+      );
+      if (customType === undefined || !isRemoteOnly(customType)) return null;
+
+      return (
+        <AssociatedDocumentsCard
```

Comment:
You're missing `isOverLimit` prop in here so they are styled differently

Reply:
How about this one?